src/tests/ftest/pool/list_verbose.py

```diff
@@ -0,0 +1,356 @@
+
+        # 7-11. Destroy and verify until the pools are gone.
+        while self.pool:
+            self.pool[-1].destroy()
+            self.pool.pop()
+
+            # Update the expected targets/scm lists too.
+            targets_disabled.pop()
+            scm_size.pop()
+            nvme_size.pop()
+
+            self.verify_pool_lists(
+                targets_disabled=targets_disabled, scm_size=scm_size,
+                nvme_size=nvme_size)
+
+    def verify_used_imbalance(self, storage):
+        """Verification steps for test_used_imbalance.
+
+        Args:
+            storage (str): NVME or SCM.
+
+        Returns:
+            list: Errors.
+
+        """
+        self.maxDiff = None
+
+        # 1. Create a pool of 80GB.
+        self.pool = []
+        if storage == "NVME":
+            self.pool.append(self.get_pool(namespace="/run/pool_both/*"))
+            nvme_size = [None]
+        else:
+            self.pool.append(self.get_pool(namespace="/run/pool_scm_only/*"))
+            nvme_size = [0]
+
+        # 2. Verify the pool created.
+        targets_disabled = [0]
+        scm_size = [None]
+        actual_pools_before = self.verify_pool_lists(
+            targets_disabled=targets_disabled, scm_size=scm_size,
+            nvme_size=nvme_size)
+
+        # 3. Store free.
+        free_before, _ = self.get_free_imbalance(
+            actual_pools_before[0], storage)
+
+        # 4, 5. Create an S1 container and run 1GB IOR with S1.
+        self.pool = self.pool[0]
+        cmd_result = self.run_ior_with_pool()
+        metrics = self.ior_cmd.get_ior_metrics(cmd_result)
+        ior_write_size = int(metrics[0][22])
+        self.log.info("IOR metrics = %s", metrics)
+        self.log.info("ior_write_size = %d", ior_write_size)
+        self.pool = [self.pool]
+
+        # 6. Verify all fields except free and imbalance. Free and imbalance are
+        # obtained from actual.
+        actual_pools_after = self.verify_pool_lists(
+            targets_disabled=targets_disabled, scm_size=scm_size,
+            nvme_size=nvme_size)
+
+        # Obtain the new free and imblanace.
+        free_after, imbalance_after = self.get_free_imbalance(
+            actual_pools_after[0], storage)
+
+        # 7, 8. Verify the free and imbalance.
+        errors = []
+        if free_after > free_before - ior_write_size:
+            errors.append(
+                "{} free is bigger than expected! Before {}; After {}".format(
+                    storage, free_before, free_after))
+
+        # Imbalance can be 10 or 11 (expecting 10% for S1 with 10GB/target pool)
+        if imbalance_after not in [10, 11]:
+            errors.append(
+                "Unexpected {} imbalance! Expected 10 or 11; Actual {}".format(
+                    storage, imbalance_after))
+
+        # 9. Destroy the container and the pool.
+        self.destroy_containers(containers=self.container)
+        self.destroy_pools(pools=self.pool)
```

Comment:
If either of theses methods fail they will return a list of reasons why they failed.  You may to fail the test if the lists are not empty.